Source/Core/Core/CoreTiming.cpp

```diff
@@ -479,8 +479,8 @@ void Idle()
 
 	idledCycles += DowncountToCycles(PowerPC::ppcState.downcount);
 	PowerPC::ppcState.downcount = 0;
-
-	Advance();
+	if(_M_ARM_32) //Temp Fix
```

Comment:
What I'm saying is that you're likely doing something that doesn't actually solve the underlying problem. All other JITs work fine with @magumagu's change. You should actually trace down *why* the JIT for ARMv7 does not work with it and fix it.